build/project-template/__PROJECT_NAME__.xcodeproj/project.pbxproj

```diff
@@ -331,6 +343,10 @@
 					"$(inherited)",
 					"DEBUG=1",
 				);
+				HEADER_SEARCH_PATHS = (
```

Comment:
We'd better leave `"$(SRCROOT)/internal" here out of compatibility reasons. "$(SRCROOT)/internal/TKLiveSync/include"` can be left deleted.